install-mac-dev.sh

```diff
@@ -9,25 +9,37 @@ if [[ $# -gt 0 ]]; then
     exit 1
 fi
 
-brew install leveldb || brew upgrade leveldb
+# Create and activate a local Python 3.6 venv
+python3.6 -m venv venv
```

Comment:
I'm not entirely sure about OpenWPM's python 2 vs 3 situation. Could you confirm whether a 3.6 venv is the correct thing to do here?
cc @englehardt

Reply:
Changed it to 3.x

Reply:
In general, Python 3(.7) seems to work, and it is the currently supported Python version.

Reply:
OpenWPM should work on 2.7 and 3.5+. Not hard coding the exact python 3 version is the right approach.